app/assets/stylesheets/application.scss

```diff
@@ -18,6 +18,8 @@ body {
   height: 100%;
 }
 
+.clearfix {clear:both;}
```

Comment:
Also, this button aligns correctly for me, although I'm not sure why everything is empty.

![screen shot 2015-07-07 at 2 03 19 pm](https://cloud.githubusercontent.com/assets/4116405/8553383/152b1ae8-24b1-11e5-8cd3-eb6ce311fdbf.png)

Perhaps the problem lies in the content of that box?  It almost seems like a closing tag is missing somewhere.